core/domain/prod_validators.py

```diff
@@ -975,514 +971,6 @@ def _get_custom_validation_functions(cls):
-        exploration category.
-
-        Args:
-            item: datastore_services.Model. GeneralSuggestionModel to validate.
-            field_name_to_external_model_references:
-                dict(str, (list(base_model_validators.ExternalModelReference))).
-                A dict keyed by field name. The field name represents
-                a unique identifier provided by the storage
-                model to which the external model is associated. Each value
-                contains a list of ExternalModelReference objects corresponding
-                to the field_name. For examples, all the external Exploration
-                Models corresponding to a storage model can be associated
-                with the field name 'exp_ids'. This dict is used for
-                validation of External Model properties linked to the
-                storage model.
-        """
-        if item.target_type not in TARGET_TYPE_TO_TARGET_MODEL:
-            cls._add_error(
-                'target %s' % base_model_validators.ERROR_CATEGORY_TYPE_CHECK,
-                'Entity id %s: Target type %s is not allowed' % (
-                    item.id, item.target_type))
-            return
-        score_category_type = (
-            item.score_category.split(
-                suggestion_models.SCORE_CATEGORY_DELIMITER)[0])
-
-        if item.target_type == suggestion_models.TARGET_TYPE_EXPLORATION:
-            target_model_references = (
-                field_name_to_external_model_references[
-                    '%s_ids' % item.target_type])
-
-            for target_model_reference in target_model_references:
-                target_model = target_model_reference.model_instance
-                if target_model is None or target_model.deleted:
-                    model_class = target_model_reference.model_class
-                    model_id = target_model_reference.model_id
-                    cls._add_error(
-                        '%s_ids %s' % (
-                            item.target_type,
-                            base_model_validators.ERROR_CATEGORY_FIELD_CHECK),
-                        'Entity id %s: based on field %s_ids having'
-                        ' value %s, expected model %s with id %s but it '
-                        'doesn\'t exist' % (
-                            item.id, item.target_type,
-                            model_id, model_class.__name__, model_id))
-
-                # Note: An exploration's category can be changed after the
-                # suggestion is submitted. Since this operation does not update
-                # the suggestion's category, we cannot assume that the
-                # exploration category matches the suggestion score category,
-                # and thus do not validate it here.
-
-        if score_category_type == suggestion_models.SCORE_TYPE_QUESTION:
-            score_category_regex = (
-                '^(%s)$' % ('|').join(VALID_SCORE_CATEGORIES_FOR_TYPE_QUESTION))
```

Comment:
It appears ``VALID_SCORE_CATEGORIES_FOR_TYPE_QUESTION`` can now be removed from this file since it was moved over.

Reply:
Done! Forgot about that after the merge, thanks for the catch